### PR TITLE
Added more courses for Data Science

### PR DESCRIPTION
**Pull request by sadafsaleem123**
## What does this PR do?
Add 10 more resource(s) | Grouped the resource(s) by websites 

## For resources
Data Science

### Description
All courses added are free.

### Why is this valuable (or not)?
Through the resource(s), the user can get more diverse idea about data science.

### How do we know it's really free?
Link is given along with their names. 

### For book lists, is it a book? For course lists, is it a course? etc.
It is a course.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by sadafsaleem123** [comment]
Checked

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -412,8 +412,10 @@
 ### Data Science
 
 * [Advanced Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/advanceddataminingwithweka/)
+* [An Introduction to Data Science](https://www.udemy.com/course/an-introduction-to-data-science/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-nw0jokvFZP8LzEEyhmdJVg&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads) - Udemy
 * [Apache Airflow Tutorials](https://www.youtube.com/playlist?list=PLYizQ5FvN6pvIOcOd6dFZu3lQqc6zBGp2) - Tuan Vu (YouTube)
 * [Applied Data Science with Python](https://cognitiveclass.ai/learn/data-science-with-python) - CognitiveClass.ai
+* [Career path for Data Scientist](https://www.mygreatlearning.com/academy/career-paths/data-scientist) - Great Learning
```
Comment (eshellman) [inline courses/free-courses-en.md:418]:
```suggestion
```
not a course

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -412,8 +412,10 @@
 ### Data Science
 
 * [Advanced Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/advanceddataminingwithweka/)
+* [An Introduction to Data Science](https://www.udemy.com/course/an-introduction-to-data-science/?ranMID=39197&ranEAID=JVFxdTr9V80&ranSiteID=JVFxdTr9V80-nw0jokvFZP8LzEEyhmdJVg&LSNPUBID=JVFxdTr9V80&utm_source=aff-campaign&utm_medium=udemyads) - Udemy
```
Comment (eshellman) [inline courses/free-courses-en.md:415]:
```suggestion
* [An Introduction to Data Science](https://www.udemy.com/course/an-introduction-to-data-science/
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -430,11 +432,18 @@
 * [Datavis 2020](https://www.youtube.com/playlist?list=PL9yYRbwpkykuK6LSMLH3bAaPpXaDUXcLV) - Curran Kelleher
 * [IBM Data Science Professional Certificate](https://www.coursera.org/professional-certificates/ibm-data-science) (Coursera)
 * [Intro to Data Analysis](https://www.udacity.com/course/intro-to-data-analysis--ud170) - Udacity
+* [Intro to Data Science](https://www.udacity.com/course/intro-to-data-science--ud359) - Udacity
+* [Introduction to Data Science](https://alison.com/course/introduction-to-data-science-revised) - Alison
+* [Introduction to Data Science](https://www.simplilearn.com/data-science-free-course-for-beginners-skillup) - SkillUp by Simplilearn
 * [Introduction to Data Science in Python](https://www.coursera.org/learn/python-data-analysis) - Christopher Brooks (Coursera)
+* [Introduction to Data Science Specialization](https://www.coursera.org/specializations/introduction-data-science) - Coursera
```
Comment (eshellman) [inline courses/free-courses-en.md:439]:
```suggestion
```
limited time course

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -430,11 +432,18 @@
...
+* [Learn Data Science](https://www.sololearn.com/learning/1093) - *registration required*
```
Comment (eshellman) [inline courses/free-courses-en.md:440]: not a direct link
```suggestion
```

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -430,11 +432,18 @@
...
+* [Learn Data Science Tutorial](https://youtu.be/ua-CiDNNj30) - (FreeCodeCamp)
```
Comment (eshellman) [inline courses/free-courses-en.md:438]:
```suggestion
* [Learn Data Science Tutorial - Full Course for Beginners](https://www.youtube.com/watch?v=ua-CiDNNj30) - Barton Poulson (FreeCodeCamp.org)
```

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -430,11 +432,18 @@
...
 * [MIT's Introduction to Computational Thinking and Data Science](https://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-0002-introduction-to-computational-thinking-and-data-science-fall-2016/) - Eric Grimson, John Guttag, Ana Bell (MIT OpenCourseWare)
 * [More Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/moredataminingwithweka/)
 * [NICO 101 - Introduction to Programming for Big Data](https://github.com/amarallab/Amaral_Lab_Intro_to_Data_Science) - Luis Amaral, Helio Tejedor, Luiz Alves
 * [The Analytics Edge](https://www.edx.org/course/analytics-edge-mitx-15-071x-3)
+* [Understanding Data Science](https://app.datacamp.com/learn/courses/data-science-for-everyone?tap_a=5644-dce66f&tap_s=1105403-195eb4&utm_medium=affiliate&utm_source=javinpaul1) - DataCamp
```
Comment (eshellman) [inline courses/free-courses-en.md:443]:
```suggestion
```
not a direct link

File: courses/free-courses-en.md [hunk 7]
```diff
@@ -1206,6 +1215,7 @@
 * [R Programming](https://www.coursera.org/course/rprog)
 * [R Programming Tutorial](https://www.youtube.com/watch?v=_V8eKsto3Ug) - Barton Poulson (freeCodeCamp)
 * [R Tutorial For Beginners \| Edureka](https://www.youtube.com/watch?v=fDRa82lxzaU) - Edureka!
+* [Statistics and R](https://pll.harvard.edu/course/statistics-and-r?delta=0) - Harward University
```
Comment (eshellman) [inline courses/free-courses-en.md:1214]:
```suggestion
```
limited time course

File: courses/free-courses-en.md [hunk 8]
```diff
@@ -412,6 +412,7 @@
 ### Data Science
 
 * [Advanced Data Mining with Weka MOOC](https://www.cs.waikato.ac.nz/ml/weka/mooc/advanceddataminingwithweka/)
+* [An Introduction to Data Science](https://www.udemy.com/course/an-introduction-to-data-science/
```
Comment (eshellman) [inline courses/free-courses-en.md:415]:
```suggestion
* [An Introduction to Data Science](https://www.udemy.com/course/an-introduction-to-data-science/)
```

**Pull request merged**